mobile/packages/ui/lib/theme/colors.dart

```diff
@@ -620,20 +620,20 @@ const EnteColorScheme darkScheme = EnteColorScheme(
 );
 
 // Background Colors
-const Color backgroundBaseLight = Color.fromRGBO(255, 255, 255, 1);
-const Color backgroundElevatedLight = Color.fromRGBO(255, 255, 255, 1);
-const Color backgroundElevated2Light = Color.fromRGBO(251, 251, 251, 1);
+const Color backgroundBaseLight = Color.fromRGBO(250, 250, 250, 1);
+const Color backgroundElevatedLight = Color.fromRGBO(0, 0, 0, 0.04);
+const Color backgroundElevated2Light = Color.fromRGBO(250, 250, 250, 1);
 
 const Color backgroundBaseDark = Color.fromRGBO(0, 0, 0, 1);
-const Color backgroundElevatedDark = Color.fromRGBO(27, 27, 27, 1);
-const Color backgroundElevated2Dark = Color.fromRGBO(37, 37, 37, 1);
+const Color backgroundElevatedDark = Color.fromRGBO(255, 255, 255, 0.04);
+const Color backgroundElevated2Dark = Color.fromRGBO(26, 26, 26, 1);
 
 // Backdrop Colors
 const Color backdropBaseLight = Color.fromRGBO(255, 255, 255, 0.92);
 const Color backdropMutedLight = Color.fromRGBO(255, 255, 255, 0.75);
 const Color backdropFaintLight = Color.fromRGBO(255, 255, 255, 0.30);
 
-const Color backdropBaseDark = Color.fromRGBO(0, 0, 0, 0.90);
+const Color backdropBaseDark = Color.fromRGBO(33, 33, 33, 0.9);
```

Comment:
Base colours should be completely opaque, same with `backgroundElevated` colours. Making them opaque will create a few issues.